Tests/function/tests/Get-VSTeamUserEntitlement.Tests.ps1

```diff
@@ -32,43 +32,116 @@ Describe "VSTeamUserEntitlement" -Tag 'VSTeamUserEntitlement' {
          BeforeAll {
             Mock _getApiVersion { return 'VSTS' }
             Mock _getInstance { return 'https://dev.azure.com/test' }
-            Mock _getApiVersion { return '1.0-unitTests' } -ParameterFilter { $Service -eq 'MemberEntitlementManagement' }
+            
 
             Mock Invoke-RestMethod { Open-SampleFile 'Get-VSTeamUserEntitlement.json' }
             Mock Invoke-RestMethod { Open-SampleFile 'Get-VSTeamUserEntitlement-Id.json' } -ParameterFilter {
                $Uri -like "*00000000-0000-0000-0000-000000000000*"
             }
          }
 
-         It 'no parameters should return users' {
-            $users = Get-VSTeamUserEntitlement
+         Context 'Get-VSTeamUserEntitlement_UpToVersion6.0' {
```

Comment:
no underscore. Should be as much readable as possible. Please also do it for the other contexts.

Reply:
Ok, I don't know why, I though that spaces was not allowed. Fixed. I will upload de changes sortly